### PR TITLE
[CVSB-14536] Fixes for Mobile App regression failures (2020-04-23)

### PR DESCRIPTION
**Pull request by jgimber-deloitte**
Updated tech records to work with the FE regression tests.

### CODE REVIEW

File: tests/resources/technical-records.json [hunk 1]
```diff
@@ -17724,7 +17724,10 @@
           "code": "v",
           "description": "3 wheelers"
         },
-        "vehicleConfiguration": "other",
+        "vehicleConfiguration": null,
```
Comment (dpanzaru-deloitte) [inline tests/resources/technical-records.json:17727]: Why was this change needed?

Comment (jgimber-deloitte) [reply tests/resources/technical-records.json]: I think I changed it as an experiment, changed back.
